spec/sidekiq_adhoc_job/requests/jobs/schedule_spec.rb

```diff
@@ -33,7 +35,7 @@
     disable_csrf
     allow(SidekiqAdhocJob::ScheduleAdhocJob)
       .to receive(:new)
-      .with('sidekiq_adhoc_job_test_dummy_worker', schedule_job_params)
+      .with('sidekiq_adhoc_job_test_dummy_worker', hash_including(schedule_job_params))
```

Comment:
Why should we change to `hash_including` here?

Reply:
because with sidekiq 8 it has `       +  "name" => "sidekiq_adhoc_job_test_dummy_worker",`

Reply:
anyway I drop the name from if/else in https://github.com/Kaligo/sidekiq_adhoc_job/pull/8/commits/1363fc68b079a8cc49d781aee4441109030713f5